apps/demo-game/src/services/PeriodResultService.ts

```diff
@@ -0,0 +1,73 @@
+import { PayloadPeriodResult, PayloadPeriodResultEnd } from '@gbl-uzh/platform'
+import { debugLog } from '@gbl-uzh/platform/dist/lib/util'
+import { produce } from 'immer'
+import { PlayerRole } from '../settings/Constants'
+import {
+  OutputPeriodResultFactsInit,
+  OutputResultFacts,
+  PeriodResultFactsInit,
+  ResultFacts,
+} from '../types/facts'
+import { PeriodFacts, PeriodSegmentFacts } from '../types/Period'
+
+const INITIAL_CAPITAL = 10000
+
+export function initialize(
+  facts: PeriodResultFactsInit,
+  payload: PayloadPeriodResult<PlayerRole, PeriodFacts>
+): OutputPeriodResultFactsInit {
+  const baseFacts: OutputPeriodResultFactsInit = {
+    resultFacts: facts,
+  }
+
+  const resultFacts: OutputPeriodResultFactsInit = produce(
+    baseFacts,
+    (draft: OutputPeriodResultFactsInit) => {
+      draft.resultFacts.decisions = {
+        bank: true,
+        bonds: false,
+        stocks: false,
+      }
+      draft.resultFacts.assets = {
+        bank: INITIAL_CAPITAL,
+        bonds: 0,
+        stocks: 0,
+        totalAssets: INITIAL_CAPITAL,
+      }
+    }
+  )
+
+  debugLog('PeriodResultInitialize', facts, payload, resultFacts)
+  return resultFacts
+}
+
+export function start(
+  facts: ResultFacts,
```

Comment:
we should make the names a bit more consistent in all files

PeriodResultFactsInit
ResultFacts

OutputPeriodResultFactsInit
OutputResultFacts

either always include the action (init, start, ...) or not?
either always include period/segment or similar in the name, or not?

Reply:
if the input is just PeriodFacts (a shared type) we could keep it with this name or reassign to a new type with consistent naming